db/schema.rb

```diff
@@ -0,0 +1,30 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
```

Comment:
you can delete innecesary comments

Reply:
this comments are generates for rails, if I eliminate it, they will reappear

Reply:
ok, then i'll approve